projects/packages/videopress/src/client/lib/resumable-file-uploader/index.ts

```diff
@@ -54,22 +57,35 @@ const resumableFileUploader = ( {
 	onError,
 }: UploadVideoArguments ) => {
 	const upload = new tus.Upload( file, {
-		onError: onError,
+		onError,
 		onProgress,
 		endpoint: tokenData.url,
 		removeFingerprintOnSuccess: true,
-		withCredentials: false,
-		autoRetry: true,
+		// withCredentials: false,
+		// autoRetry: true,
```

Comment:
Not needed?

Reply:
Actually no use for those. There are no such options on the underlying lib